.github/workflows/minikube.yml

```diff
@@ -0,0 +1,172 @@
+---
+name: MiniKube Heml Deployment
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  job1:
+    runs-on: ubuntu-latest
+    name: build and deploy to minikube
+    steps:
+      - uses: actions/checkout@v2
+
+      # HELM Config
+      - name: Install Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.10.0
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+
+      - run: mkdir logs
+
+      # Install dependencies
+      - run: npm ci
+
+      # Create certs
+      - run: sudo chown -R runner:docker certs
+      - run: npm run certs
+
+      # Remove route set for openshift
+      - name: remove route config
+        run: sudo rm helm/bailo/templates/route.yaml
+
+      # Add bailo NodePort
+      - name: Create NodePort Config
+        run: |
+          cat << EOT >> helm/bailo/templates/nodePort.yaml
+          ---
+          apiVersion: v1
+          kind: Service
+          metadata:
+            name: bailo-node-port
```

Comment:
Don't hardcode bailo, instead use the variable.

Reply:
ref to bailo now removed. This is temp file created only with action is run.